applications/StructuralMechanicsApplication/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ message("**** configuring KratosStructuralMechanicsApplication ****")
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
 include_directories( ${CMAKE_SOURCE_DIR}/applications/StructuralMechanicsApplication )
 include_directories( ${CMAKE_SOURCE_DIR}/applications/MeshingApplication )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/MappingApplication )
```

Comment:
is this still necessary?

Reply:
No I will remove it in further commits. Thanks for noticing .

Reply:
ping